routing/http/contentrouter/contentrouter.go

```diff
@@ -3,55 +3,44 @@ package contentrouter
 import (
 	"context"
 	"reflect"
-	"time"
+	"strings"
 
-	"github.com/ipfs/boxo/routing/http/internal"
+	"github.com/ipfs/boxo/ipns"
 	"github.com/ipfs/boxo/routing/http/types"
 	"github.com/ipfs/boxo/routing/http/types/iter"
 	"github.com/ipfs/go-cid"
 	logging "github.com/ipfs/go-log/v2"
+	routinghelpers "github.com/libp2p/go-libp2p-routing-helpers"
 	"github.com/libp2p/go-libp2p/core/peer"
 	"github.com/libp2p/go-libp2p/core/routing"
 	"github.com/multiformats/go-multiaddr"
 	"github.com/multiformats/go-multihash"
 )
 
-var logger = logging.Logger("service/contentrouting")
-
-const ttl = 24 * time.Hour
+var logger = logging.Logger("routing/http/contentrouter")
 
 type Client interface {
-	ProvideBitswap(ctx context.Context, keys []cid.Cid, ttl time.Duration) (time.Duration, error)
```

Comment:
@lidel I decided to Go with A, even though I would rather prefer a B. I marked a bunch of types as deprecated, and now the CI is complaining because we use them ourselves 😭. I'm gonna see if I can "fix" that.

Reply:
Thanks. Filled https://github.com/ipni/index-provider/issues/403 so we don't forget to update it when the IPIP lands.